packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -311,16 +311,10 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
       );
       break;
     /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
-    case holdPageTrampoline():
+    case 'hold':
       pageElement = (
         <>
```

Comment:
Why are we changing hold page logic here?

Reply:
https://github.com/Azure/communication-ui-library/pull/2905
it's coming from this, need to add new CC feature to make this work, this part is to adjust to that new CC feature